Ginger/GingerCoreNET/Application Models/Learn/POM/PomLearnUtils.cs

```diff
@@ -144,7 +145,11 @@ public void ClearStopLearning()
 
         public void PrepareLearningConfigurations()
         {
-            SelectedElementTypesList = AutoMapElementTypesList.Where(x => x.Selected == true).Select(x => x.ElementType).ToList();
+            var uIElementList = new List<UIElementFilter>();
+            uIElementList.AddRange(AutoMapBasicElementTypesList.ToList());
+            uIElementList.AddRange(AutoMapAdvanceElementTypesList.ToList());
+
+            SelectedElementTypesList = uIElementList.Where(x => x.Selected == true).Select(x => x.ElementType).ToList();
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5763742)